packages/amplication-data-service-generator/src/server/resource/resolver/resolver.base.template.ts

```diff
@@ -3,7 +3,7 @@ import * as graphql from "@nestjs/graphql";
 import * as apollo from "apollo-server-express";
 import * as nestAccessControl from "nest-access-control";
 // @ts-ignore
-import * as gqlDefaultAuthGuard from "../../auth/gqlDefaultAuth.guard";
+import { GqlDefaultAuthGuard } from "../../auth/gqlDefaultAuth.guard";
```

Comment:
For any future case, please do not include such changes with other changes. 
Any code refactoring especially ones that impact the code generation should be justified and included in a separate PR. 
for this case, just leave it as it is.